src/lang/Messages.properties

```diff
@@ -71,6 +71,7 @@ api.dialogue.browseApiNotEnabled.title = API Not Enabled
 api.dialogue.browseApiNotEnabled.message = The API must be enabled before you can browse it.\nDo you want to enable the API?
 api.dialogue.browseApiNotEnabled.button.confirm.label = Enable and Browse the API
 api.dialogue.browseApiNotEnabled.button.cancel.label = Cancel
+api.deprecated.option.endpoint = Option no longer in effective use.
```

Comment:
That's what I meant, those API endpoints can still be used but they shouldn't (either are not working properly or were replaced by better alternatives), also, the corresponding (Java) methods are no longer in use in ZAP (i.e. no ZAP code is calling those methods).